pkg/rtcconfig/webrtc_config_test.go

```diff
@@ -0,0 +1,78 @@
+package rtcconfig
+
+import (
+	"net"
+	"testing"
+)
+
+func Test_IPFilterFromConf(t *testing.T) {
+	testData := IPsConfig{
+		Includes: []string{"10.0.0.0/19"},
+		Excludes: []string{"10.0.0.0/9", "10.192.0.0/11", "10.244.0.0/16", "172.16.0.0/12", "192.168.128.0/17"},
+	}
+
+	ipFilter, err := IPFilterFromConf(testData)
+
+	if err != nil {
```

Comment:
wanna use `require.NoError` for consistency with our other tests? that would help to fail the test as well.